docs/api/api/api.txt

```diff
@@ -1995,8 +1995,8 @@ POST /staging/<staging_workflow_project>/staging_projects/<staging_project>/stag
 XmlBody: remove_staged_requests
 XmlResult: status_ok
 
-DELETE /staging/<staging_workflow_project>/staging_projects/<staging_project>/staged_requests
-  Remove one or multiple bs_requests to the staged requests of a staging project.
+DELETE /staging/<staging_workflow_project>/staged_requests
+  Remove one or multiple bs_requests from the staged requests of a staging project.
```

Comment:
> Remove one or multiple bs_requests from the staged requests of a staging project.

Remove requests from the staging projects they are staged in.

Reply:
why is it saying bs_request anyway? I'm painfully aware why the class is named this way, but in api documentation it should be request, no?